doc/api/dgram.md

```diff
@@ -70,19 +70,21 @@ datagram messages. This occurs as soon as UDP sockets are created.
 added: v0.1.99
 -->
 
+The `'message'` event is emitted when a new datagram is available on a socket.
+The event handler function is passed two arguments: `msg` and `rinfo`.
 * `msg` {Buffer} - The message
 * `rinfo` {Object} - Remote address information
-
-The `'message'` event is emitted when a new datagram is available on a socket.
-The event handler function is passed two arguments: `msg` and `rinfo`. The
-`msg` argument is a [`Buffer`][] and `rinfo` is an object with the sender's
-address information provided by the `address`, `family` and `port` properties:
+  * `address` {String} The sender's address
+  * `family` {String} The address's family
```

Comment:
should say what the address family string can be, `'IPv4'` or `'IPv6'`, I think? Its not easy to guess. The example below shows the IPv4 output, but in some places in the API various protocol type identifiers are not handled uniformly